From 2447d52d6a8178a21067ae7e7e11c4c361d1f665 Mon Sep 17 00:00:00 2001
From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 10:53:52 -0700
Subject: 01_confiure

Remove the extra library added by the configure.py script. Doesn't fit
the Debian library naming
Patch-Name: 01_confiure.patch
---
 configure/configure.py | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/configure/configure.py b/configure/configure.py
index ca560b3..ea7978b 100755
--- a/configure/configure.py
+++ b/configure/configure.py
@@ -768,6 +768,9 @@ def setup_qwt5_build(configuration, options, package):
     elif qwt not in options.extra_libs:
         # no, add the qwt library if needed
         options.extra_libs.append(qwt)
+    # Added for the Debian packaging script
+    while options.extra_libs.count(qwt):
+        options.extra_libs.remove(qwt)
 
     # zap the temporary directory
     try:
@@ -1119,9 +1122,9 @@ def main():
 if __name__ == '__main__':
     try:
         main()
-    except Die as e:
-        inform('%s\n' % e.args)
-        sys.exit(1)
+    #except Die as e:
+    #    inform('%s\n' % e.args)
+    #    sys.exit(1)
     except:
         for entry in traceback.extract_tb(sys.exc_info()[-1]):
             if 'optparse.py' in entry[0]:
